plugin/trino-postgresql/src/main/java/io/trino/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -1529,7 +1529,7 @@ private static ColumnMapping typedVarcharColumnMapping(String jdbcTypeName)
 
     private static SliceWriteFunction typedVarcharWriteFunction(String jdbcTypeName)
     {
-        String bindExpression = format("CAST(? AS %s)", requireNonNull(jdbcTypeName, "jdbcTypeName is null"));
+        String bindExpression = format("CAST(? AS \"%s\")", requireNonNull(jdbcTypeName, "jdbcTypeName is null"));
```

Comment:
This is used by json and jsonb as well ? Can we have test coverage for the same.

Reply:
They are built-in types. What tests are you recommending?

Reply:
I guess TypeMapping test would cover for it.